pkg/quay/quay.go

```diff
@@ -401,3 +403,79 @@ func handleRobotName(robotName string) (string, error) {
 	}
 	return robotName, nil
 }
+
+func (c *QuayClient) GetTagsFromPage(organization, repository string, page int) ([]Tag, bool, error) {
+	url := fmt.Sprintf("%s/repository/%s/%s/tag/?page=%d", c.url, organization, repository, page)
+
+	req, err := http.NewRequest(http.MethodGet, url, nil)
+	if err != nil {
+		return nil, false, fmt.Errorf("failed to create new request, error: %s", err)
+	}
+
+	req.Header.Add("Authorization", fmt.Sprintf("Bearer %s", c.AuthToken))
+	req.Header.Add("Content-Type", "application/json")
+
+	res, err := c.httpClient.Do(req)
+	if err != nil {
+		return nil, false, fmt.Errorf("failed to Do request, error: %s", err)
+	}
+	if res.StatusCode != 200 {
+		return nil, false, fmt.Errorf("error getting repositories, got status code %d", res.StatusCode)
+	}
+
+	defer res.Body.Close()
+	body, err := io.ReadAll(res.Body)
+	if err != nil {
+		return nil, false, fmt.Errorf("failed to read response body, error: %s", err)
+	}
+
+	var response struct {
+		Tags          []Tag
+		Page          int
+		HasAdditional bool
```

Comment:
I think these fields should have json struct tags, at least definitely `HasAdditional`
```go
HasAdditional bool `json:"has_additional"`
```
otherwise the response is not unmarshalled correctly